File: API/Entities/Product.cs
namespace API.Entities
{
    public class Product
    {
        public int Id { get; set; }
        
        public string Prod_name { get; set; }
        public string Prod_desc { get; set; }
        public long Price { get; set; }
        public string Prod_pics { get; set; }
        public string Prod_type { get; set; }
        public string Brand { get; set; }
        public int QuantityInStock { get; set; }
    }
}

File: API/client/src/app/models/product.ts
export interface Product {
    id: number
    prod_name: string
    prod_desc: string
    price: number
    prod_pics: string
    prod_type?: string
    brand: string
    quantityInStock?: number
}
// ? = optional

export interface ProductParams {
    orderBy: string;
    searchTerm?: string;
    types: string[];
    brands: string[];
    pageNumber: number;
    pageSize: number;
}